Test/git-issues/git-issue-4012.dfy

```diff
@@ -1,7 +1,7 @@
-// RUN: %testDafnyForEachCompiler "%s" -- --relax-definite-assignment
```

Comment:
I'm confused because this PR says it fixes issue #4125, and in that issue the reproduction example given is:

```
method Main() {
    var x: multiset<int> := multiset{};
    x := x[1 := 18446744073709551616];
    print |x|, "\n";
}
```

But apparently that example was already in our test-suite. How come it was not triggering the bug?

Reply:
Robin checked in `Test/git-issues/git-issue-4012.dfy.cs.check`, which grants an exemption to the tester. At the time I only fixed it for Python.